### PR TITLE
async :)

### PR DESCRIPTION
**Pull request by samirans89**
- Successful test [build](https://automate.browserstack.com/dashboard/v2/builds/55ef4160a81fffee25d68ebe6e49405e4eedeb8b)
- Same tests already working on Android mobile.
- WDIO Types defined for custom commands
- Mochawesome report + Allure
- GitHub Actions

### CODE REVIEW

File: __tests__/pages/checkoutPage.ts [hunk 1]
```diff
@@ -3,43 +3,37 @@ import { Page } from "./basePage";
  * sub page containing specific selectors and methods for a specific page
  */
 export class CheckoutPage extends Page {
-  /**
-   * define selectors using getter methods
-   */
-  get loginText(): WebdriverIO.Element {
```
Comment (abhi-singhs) [inline __tests__/pages/checkoutPage.ts:9]: Any particular reason for removing the getter method?

File: __tests__/suites/paypal-button-click-pom.test.ts [hunk 2]
```diff
@@ -2,26 +2,22 @@ import { expect } from "chai";
 import { CheckoutPage } from "../pages/checkoutPage";
 import { HomePage } from "../pages/homePage";
 
-describe("Testing Paypal buttons", () => {
-  beforeEach("Open StackDemo", () => {
-    browser.url("/demo/checkout/#/pattern/client");
+describe("Testing Paypal buttons", async () => {
```
Comment (NightWing1998) [inline __tests__/suites/paypal-button-click-pom.test.ts]: no need for an async with describe. Can we confirm if this is hampering functionality in any way?

File: resources/config/wdio.prod.browserstack.conf.ts [hunk 3]
```diff
@@ -3,78 +3,99 @@ import * as _ from "lodash";
...
     }
   },
 };
 
 export const config = _.defaultsDeep(overrides, defaultConfig);
+
+config.capabilities.forEach(function (caps: { [x: string]: unknown }) {
+  for (const i in config.commonCapabilities)
+    caps[i] = caps[i] || config.commonCapabilities[i];
+});
```
Comment (NightWing1998) [inline resources/config/wdio.prod.browserstack.conf.ts]: can we refactor this? It looks weird to export something and then change its value

File: __tests__/suites/paypal-button-click-pom.test.ts (hunk 2 above)
Comment (samirans89) [reply __tests__/suites/paypal-button-click-pom.test.ts]: done. it wont impact functionality. 
the wdio docs define it without async so we are good.
https://webdriver.io/docs/sync-vs-async/

File: resources/config/wdio.prod.browserstack.conf.ts (hunk 3 above)
Comment (samirans89) [reply resources/config/wdio.prod.browserstack.conf.ts]: updated.

**Pull request merged**